<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LuaWindow</class>
 <widget class="QMainWindow" name="LuaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>790</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lua editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QTreeView" name="OpenedFilesList">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QTabWidget" name="tabWidget"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QTreeView" name="StatusTreeView">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_script"/>
   <addaction name="actionOpenScript"/>
   <addaction name="actionSaveScript"/>
   <addaction name="separator"/>
   <addaction name="actionCloseScript"/>
   <addaction name="actionClose_all_scipts"/>
   <addaction name="separator"/>
   <addaction name="actionSendScript"/>
   <addaction name="actionKill_script_instances"/>
   <addaction name="separator"/>
   <addaction name="actionReload_script"/>
   <addaction name="actionReload_all_scripts"/>
   <addaction name="separator"/>
   <addaction name="actionValidate_script"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenScript">
   <property name="icon">
    <iconset resource="../mgdt.qrc">
     <normaloff>:/mgdt/icons/yellow_open.png</normaloff>:/mgdt/icons/yellow_open.png</iconset>
   </property>
   <property name="text">
    <string>Open script</string>
   </property>
   <property name="toolTip">
    <string>Open scipt file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveScript">
   <property name="icon">
    <iconset resource="../mgdt.qrc">
     <normaloff>:/mgdt/icons/blue_down_save.png</normaloff>:/mgdt/icons/blue_down_save.png</iconset>
   </property>
   <property name="text">
    <string>Save script</string>
   </property>
   <property name="toolTip">
    <string>Save scipt to file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSendScript">
   <property name="icon">
    <iconset resource="../mgdt.qrc">
     <normaloff>:/mgdt/icons/blue_top_left_arrow.png</normaloff>:/mgdt/icons/blue_top_left_arrow.png</iconset>
   </property>
   <property name="text">
    <string>Send script</string>
   </property>
   <property name="toolTip">
    <string>Send script (whole or selected part) to MoonGlare</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionCloseScript">
   <property name="icon">
    <iconset resource="../mgdt.qrc">
     <normaloff>:/mgdt/icons/red_minus.png</normaloff>:/mgdt/icons/red_minus.png</iconset>
   </property>
   <property name="text">
    <string>Close script</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSendSelctedText">
   <property name="text">
    <string>Send selcted text</string>
   </property>
   <property name="toolTip">
    <string>Send selected text to MoonGlare engine</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionReload_script">
   <property name="icon">
    <iconset resource="../mgdt.qrc">
     <normaloff>:/mgdt/icons/blue_circle_left_arrow.png</normaloff>:/mgdt/icons/blue_circle_left_arrow.png</iconset>
   </property>
   <property name="text">
    <string>Reload script</string>
   </property>
  </action>
  <action name="actionKill_script_instances">
   <property name="icon">
    <iconset resource="../mgdt.qrc">
     <normaloff>:/mgdt/icons/red_cross.png</normaloff>:/mgdt/icons/red_cross.png</iconset>
   </property>
   <property name="text">
    <string>Kill script instances</string>
   </property>
  </action>
  <action name="actionNew_script">
   <property name="icon">
    <iconset resource="../mgdt.qrc">
     <normaloff>:/mgdt/icons/blue_file.png</normaloff>:/mgdt/icons/blue_file.png</iconset>
   </property>
   <property name="text">
    <string>New script</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionValidate_script">
   <property name="icon">
    <iconset resource="../mgdt.qrc">
     <normaloff>:/mgdt/icons/green_ok.png</normaloff>:/mgdt/icons/green_ok.png</iconset>
   </property>
   <property name="text">
    <string>Validate script</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionReload_all_scripts">
   <property name="icon">
    <iconset resource="../mgdt.qrc">
     <normaloff>:/mgdt/icons/blue_down_arrow.png</normaloff>:/mgdt/icons/blue_down_arrow.png</iconset>
   </property>
   <property name="text">
    <string>Reload all scripts</string>
   </property>
  </action>
  <action name="actionClose_all_scipts">
   <property name="text">
    <string>Close all scipts</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../mgdt.qrc"/>
 </resources>
 <connections/>
</ui>
